content/en/tracing/profiler/compare_profiles.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Compare Profiles
+kind: documentation
+further_reading:
+    - link: 'tracing/profiler/enabling'
+      tag: 'Documentation'
+      text: 'Enable continuous profiler for your application'
+    - link: 'getting_started/profiler'
+      tag: 'Documentation'
+      text: 'Getting Started with Profiler'
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tag: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog'
+---
+
+Continuous Profiler can compare two profiles or profile aggregations with each other to help you identify code performance improvements, regressions, and structural changes. You can compare a profile with:
+
+- The same profile at a different time,
+- An average of the service's profiles over time, or
+- A profile with a different set of Datadog tags (for example, environment, version, or data center). 
+
+This helps you see if the service is taking more time, using more memory, making more allocations, throwing more exceptions, or involving more or less code and calls than it was in the past.
+
+## Comparison scenarios
+
+Comparisons work best when the application is experiencing a similar workload (total requests and resource usage) as it was in the past.
+
+Some common scenarios to use comparison are:
+
+- Comparing two latest deployments. For example, verify if the latest deployed fix lowers the number of memory allocations your method makes.
+
+- Comparing two distinct time periods. For example, calculate the CPU consumption for today compared to the past week. What methods get better or worse in terms of CPU consumption?
+
+- Comparing two different sets of tags. For example, compare profiles between different environments, availability zones, pods, canaries, or other custom Datadog tags.
+
+## Accessing the comparison view
+
+You can open different types of comparisons from different places in the UI. 
+
+### Comparing a profile to a previous time period 
+
+On the Profiler Search view, select a profile from the list. Click **Compare** to open the comparison view. By default, the selected profile is shown as Profile B. For Profile A, select an aggregation time frame and tags, or a specific profile ID. 
+
+Select the metric you want to compare (the list varies based on code language). This can be helpful, for example, for looking at allocation spikes while investigating CPU profiles.
+
+{{< img src="tracing/profiling/compare_time_frames.mp4" alt="Opening the comparison view to compare a profile with a aggregation for a time frame." video="true">}}
+
+Take note of the legend colors, which show:
+ - Deepening shades of red for methods that are slower in profile B.
```

Comment:
Lets replace faster and slower with "takes less time" and "takes more time" since technically those methods could be running less often during your comparison time period.

Reply:
```suggestion
 - Deepening shades of red for methods that takes more time in profile B.
```